instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -349,10 +349,7 @@ def test_trace_response_headers(self):
         )
         self.assertEqual(
             response.headers["traceresponse"],
-            "00-{0}-{1}-01".format(
```

Comment:
Cases like these where you don't have short variable, f-strings can be pretty ugly :(

Reply:
I hear that... I don't feel strongly either ways. I found the formatting previously there confusing as well w/ the multiple 0's and 1's 😄 

Could create a temporary variable or just disable pylint for this line.

Reply:
It's probably more readable when it's not complex code buried in a string - I prefer the format one.

Reply:
> I found the formatting previously there confusing as well w/ the multiple 0's and 1's

I agree, maybe short variable names with f-strings would be better here, or could do `"00-{}-{}-01".format(...)` or named format params. I'll leave it up to you, don't want to block this @codeboten

Reply:
i'm happy to fix this in a following PR. I will refactor all the tests that are duplicated into a single test in the base class.

Reply:
will use the following method to simplify the code https://github.com/open-telemetry/opentelemetry-python/pull/2159